rubocop/defaults-0.49.1.yml

```diff
@@ -223,7 +223,6 @@
 - Style/BeginBlock
 - Style/BlockComments
 - Style/BlockDelimiters
-- Style/BracesAroundHashParameters
```

Comment:
This was not enough to stop the cop being *`enabled`* everywhere. See https://github.com/puppetlabs/pdk-templates/pull/344